stream-chat-android-sample/src/main/res/layout/fragment_login.xml

```diff
@@ -1,32 +1,156 @@
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/summaryTextView" />
+
+        <EditText
+            android:id="@+id/apiKeyEditText"
+            style="@style/LoginEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/login_field_api_key_hint"
+            tools:ignore="Autofill" />
+
+        <EditText
+            android:id="@+id/userIdEditText"
+            style="@style/LoginEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/login_field_user_id_hint"
+            tools:ignore="Autofill" />
+
+        <EditText
+            android:id="@+id/userTokenEditText"
+            style="@style/LoginEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/login_field_user_token"
+            android:singleLine="false"
+            android:inputType="textMultiLine"
+            tools:ignore="Autofill" />
+
+        <EditText
+            android:id="@+id/userNameEditText"
+            style="@style/LoginEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/login_field_user_name_hint"
+            android:imeOptions="actionDone"
+            tools:ignore="Autofill" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/loginButton"
+            style="@style/Widget.MaterialComponents.Button.UnelevatedButton"
+            android:layout_width="match_parent"
+            android:layout_height="48dp"
+            android:layout_marginBottom="4dp"
+            android:insetTop="0dp"
+            android:insetBottom="0dp"
+            android:text="@string/login_button_text"
+            android:textAppearance="@style/text"
+            android:textSize="16sp"
+            app:cornerRadius="@dimen/login_controls_corner_radius"
+            app:layout_constraintBottom_toTopOf="@id/developmentUsers"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent" />
+
+        <TextView
+            android:id="@+id/developmentUsers"
+            android:layout_width="match_parent"
+            android:layout_height="48dp"
+            android:background="?selectableItemBackground"
+            android:gravity="center"
+            android:text="@string/login_development_users_text"
+            android:textColor="@color/colorPrimary"
+            android:textSize="18sp"
+            app:layout_constraintBottom_toTopOf="@id/sdkVersion"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent" />
```

Comment:
This button on the UI feels a bit hidden to me. For devs taking a first look at the SDK, it might be intimidating to get the form with a ton of fields requiring keys and stuff, and they might not bother trying out our app because of that.

I don't know if we've already made these design considerations, but maybe we could have a more balanced screen, maybe a quick choice like "I just want to try the sample quickly" and "I already have an API key, let me try the app with that".

Reply:
Lets leave it as is for now, the design is about to change anyway